packages/block-library/src/image/edit.native.js

```diff
@@ -254,9 +255,10 @@ class ImageEdit extends React.Component {
 		const toolbarEditButton = (
 			<Toolbar>
 				<ToolbarButton
-					label={ __( 'Edit image' ) }
+					title={ __( 'Edit image' ) }
 					icon="edit"
 					onClick={ onMediaOptionsButtonPressed }
+					extraProps={ { hint: __( 'Double tap to edit image' ) } }
```

Comment:
I don't think that the hint here is adding new information, so it's probably not needed.